src/engines/nwn2/roster.h

```diff
@@ -57,11 +60,12 @@ class Roster {
 
 private:
 	struct Member {
-		Member() : rosterName(""),
+		Member() : rosterName(""),      cTemplate(""),
```

Comment:
Common::UString already constructs to empty, so you don't really need to do this here as well. (But it doesn't break anything either).

For another C++11-ism, you could inline brace-initializers for these member variables instead of doing that in a constructor. I.e. instead of
```
		bool isAvailable;           ///< Not currently in a party?
```
do
```
		bool isAvailable { false };           ///< Not currently in a party?
```

Again, not strictly necessary.

Reply:
Understood. I think I'll leave it there though so it's known it was intentional. Thanks.